File: Mobile/nomex/lib/Models/AuthRequest.dart
import 'package:json_annotation/json_annotation.dart';

part 'AuthRequest.g.dart';

@JsonSerializable()
class AuthRequest{

  String? name;
  String? surname;
  String email;
  String password;

  AuthRequest(this.email, this.password);

  factory AuthRequest.fromJson(Map<String, dynamic> json) => _$AuthRequestFromJson(json);

  Map<String, dynamic> toJson() => _$AuthRequestToJson(this);

}

File: API/Nomex/Nomex/Auth/AuthRequest.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Nomex.Auth
{
    public class AuthRequest
    {
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
    }
}
